src/components/mapPageComponents/ducks/types.ts

```diff
@@ -0,0 +1,68 @@
+import { C4CState } from '../../../store';
+import { ThunkAction } from 'redux-thunk';
+import { MapActions } from './actions';
+import { ApiExtraArgs } from '../../../api/apiClient';
+import { AsyncRequest } from '../../../utils/asyncRequest';
+
+// ---------------------------------Blocks----------------------------------------
+
+export interface BlockGeoData {
+  type: string;
+  name: string;
+  features: BlockFeatureResponse[];
+}
+
+interface BlockFeatureResponse {
+  type: string;
+  properties: BlockFeaturePropertiesResponse;
+  geometry: MapGeometry;
+}
+
+interface BlockFeaturePropertiesResponse {
+  block_id: number; // does not match linting
```

Comment:
For all these variables, we should use lowerCamelCase on the frontend, and reassign the object values in the api client before returning to the rest of the application.